fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTableSchema.java

```diff
@@ -35,7 +37,8 @@ public PhysicalTableSchema(
         super(timeGrain, columns);
         this.timeGrain = timeGrain;
 
-        this.logicalToPhysicalColumnNames = Collections.unmodifiableMap(logicalToPhysicalColumnNames);
+        this.logicalToPhysicalColumnNames = Collections.unmodifiableMap(new LinkedHashMap<>
+                (logicalToPhysicalColumnNames));
```

Comment:
Unwrap? Or wrap at the start of the param list for `Collections.unmodifiableMap`